frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/common-template-validations/selectors.ts

```diff
@@ -0,0 +1,19 @@
+import { iGetIn } from '../../../../../utils/immutable';
+import { CommonTemplatesValidation } from './validation-types';
+
+export const getValidationsFromTemplates = (templates): CommonTemplatesValidation[] => {
+  return templates
+    .map((relevantTemplate) =>
+      JSON.parse(iGetIn(relevantTemplate, ['metadata', 'annotations', 'validations'])),
+    )
+    .valueSeq()
```

Comment:
question: what does `.valueSeq().toArray()` adds here ? does it work if we remove them ?

Reply:
it gets the validations as an array from the immutable object. Thus cannot be removed.